packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -31,7 +31,7 @@ class AnalyzeCommand extends FlutterCommand {
   @override
   Future<int> runInProject() async {
     Set<String> pubSpecDirectories = new Set<String>();
-    List<String> dartFiles = argResults.rest.toList();
+    Set<String> dartFiles = new Set<String>.from(argResults.rest);
```

Comment:
I think if we ever have a duplicate, it should be an error and we should fix it.